### DISCUSSION TITLE
Lazy loading failing

### DISCUSSION THREAD

**Discussion by paulvonber** [nuxt-modules/i18n#1953] [Q&A]
I don't fully understand what is wrong with lazy loading, by following the documentation:
https://v8.i18n.nuxtjs.org/guide/lazy-load-translations

I get the following error:
<img width="612" alt="image" src="https://user-images.githubusercontent.com/37245111/227714642-d0d0df6f-dabf-45ba-a724-bccd72c1ee7e.png">

is there any suggestions what exactly I'm doing wrong? or it's just doesn't work yet?
Thanks

**Comment by Eddie344** [comment]
I have the same problem
[Issue](https://github.com/nuxt-modules/i18n/issues/1954)
